osf/models/node.py

```diff
@@ -441,12 +441,39 @@ def is_preprint(self):
         if not self.preprint_file_id or not self.is_public:
             return False
         if self.preprint_file.node_id == self.id:
-            return self.has_published_preprint
+            return self.has_preprint_non_initial_state
         else:
             self._is_preprint_orphan = True
             return False
 
     @property
+    def has_preprint_non_initial_state(self):
```

Comment:
I think it'd be clearer and more consistent to call this `has_submitted_preprint`, or some other `has_<adjective>_preprint`